src/reducers/versions.spec.tsx

```diff
@@ -1589,12 +1584,9 @@ describe(__filename, () => {
       diff: ExternalDiff | null,
     ): ExternalVersionWithDiff => {
       return {
-        ...fakeVersionWithContent,
+        ...fakeVersionWithDiff,
         file: {
-          ...fakeVersionWithContent.file,
-          base_file: {
-            id: nextUniqueId(),
-          },
+          ...fakeVersionWithDiff.file,
```

Comment:
these two changes from `fakeVersionWithContent` to `fakeVersionWithDiff` - are they related to removing `base_file` too?  I'm not overly familiar with this code but it seems unrelated.

Reply:
No, that was a drive-by fix that I noticed should be done while I was updating this test code.